stag-library-compiler/src/test/java/com/vimeo/stag/processor/TypeUtilsUnitTest.kt

```diff
@@ -285,113 +309,40 @@ class TypeUtilsUnitTest : BaseUnitTest() {
-        Assert.assertTrue(TypeUtils.isSupportedNative(Byte::class.javaPrimitiveType!!.name))
-        Assert.assertTrue(TypeUtils.isSupportedNative(Char::class.javaPrimitiveType!!.name))
-        Assert.assertTrue(TypeUtils.isSupportedNative(Short::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Long::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Int::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Boolean::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Float::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Double::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Byte::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Char::class.javaPrimitiveType!!.name))
+        assertTrue(TypeUtils.isSupportedPrimitive(Short::class.javaPrimitiveType!!.name))
 
         // Check unsupported primitives
-        Assert.assertFalse(TypeUtils.isSupportedNative(Void.TYPE.name))
+        assertFalse(TypeUtils.isSupportedPrimitive(Void.TYPE.name))
 
         // Check non-primitives
-        Assert.assertFalse(TypeUtils.isSupportedNative(Any::class.java.name))
+        assertFalse(TypeUtils.isSupportedPrimitive(String::class.java.name))
+        assertFalse(TypeUtils.isSupportedPrimitive(Any::class.java.name))
     }
 
     @Test
     @Throws(Exception::class)
     fun testIsMap_supportsCorrectTypes() {
         // Check null
-        Assert.assertFalse(TypeUtils.isSupportedMap(null))
+        assertFalse(TypeUtils.isSupportedMap(null))
 
         // Check supported types
-        Assert.assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(Map::class.java)))
-        Assert.assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(HashMap::class.java)))
-        Assert.assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(LinkedHashMap::class.java)))
+        assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(Map::class.java)))
+        assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(HashMap::class.java)))
+        assertTrue(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(LinkedHashMap::class.java)))
 
         // Check type that implements map
-        Assert.assertFalse(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(DummyMapClass::class.java)))
+        assertFalse(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(DummyMapClass::class.java)))
 
         // Check other types
-        Assert.assertFalse(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(Any::class.java)))
-        Assert.assertFalse(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(DummyConcreteClass::class.java)))
-        Assert.assertFalse(TypeUtils.isSupportedMap(Utils.getTypeMirrorFromClass(String::class.java)))
```

Comment:
I like this removal of `Assert.`. There are also 4 remaining `Assert.assertXXXX` in SwappableParserExampleModelTest.java that could also be changed if you want to convert them all while you're at it.